ui/tract-renderer/src/main/java/org/cru/godtools/tract/analytics/model/ContentAnalyticsActionEvent.kt

```diff
@@ -10,20 +9,5 @@ class ContentAnalyticsActionEvent(@VisibleForTesting internal val event: Analyti
     AnalyticsActionEvent(action = event.action.orEmpty()) {
```

Comment:
This is currently not working since the event names are still adobe based, but once the tools XML is updated to firebase it will work.

Reply:
thankfully `firebaseEventName` on this event will correctly sanitize the names when the analytics event was for Adobe originally.